core/api/views/business_plan.py

```diff
@@ -79,6 +79,33 @@ def get_years(self, *args, **kwargs):
             )
         )
 
+    @action(methods=["GET"], detail=False)
+    def get(self, *args, **kwargs):
```

Comment:
I think is good to have some tests for it. It is an important endpoint and we need to have it secure.